### PR TITLE
Carousel

### PR DESCRIPTION
**Pull request by mikeheft**
Additions: 
Add path helper

User Story:
1

Concerns (code location):
`welcome#index`

Mentions to collaborators:
@amhursh @blsrofe @JunePaloma 

Notes(if applicable):
Fixed path helper to now properly display category name in url

### CODE REVIEW

**Review by blsrofe — COMMENTED** [review]
Good job! Looks like we have category_slug.rb and slug_creator.rb doing the same thing. Was that intentional or did you just forget to delete one?

**Comment by mikeheft** [comment]
It was a goof on my part, good catch, I'll delete that and push that change. I misunderstood the url changes and thought I needed to create a slug for the category as well. But I fixed that. 😄

**Comment by mikeheft** [comment]
@blsrofe @amhursh @JunePaloma I removed the extra files. This should be good now.

File: app/services/store_creator.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  end
+
+  private
+    attr_reader :name,
+                :description
+
+    def slug
+      name.parameterize
+    end
+end
```
Comment (amhursh) [inline app/services/store_creator.rb:18]: nice clean poro for the slug. looks good.

File: config/routes.rb [hunk 2]
```diff
@@ -29,10 +29,10 @@
   get '/cart', to: "carts#index", as: "cart"
   delete '/cart/:id', to: "carts#destroy", as: "cart_remove_item"
   put '/cart/:id', to: "carts#update", as: "cart_add_item"
-  get '/:category_name', to: 'categories#show'
 
   post '/orders/cancel/:order_id', to: "orders#cancel", as: "order_cancel"
   post '/orders/paid/:order_id', to: "orders#paid", as: "order_paid"
   post '/orders/completed/:order_id', to: "orders#completed", as: "order_completed"
+  get '/:category_name', to: 'categories#show'
```
Comment (amhursh) [inline config/routes.rb:36]: Ah good. I do think this should be at the end.

**Review by amhursh — APPROVED** [review]

**Pull request merged**